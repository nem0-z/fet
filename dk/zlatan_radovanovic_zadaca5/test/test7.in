print "Temperatura je " & (20 * 9/5 + 32)  & " Fahrenheita" 
