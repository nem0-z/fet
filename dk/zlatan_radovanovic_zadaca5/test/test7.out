Temperatura je 68.000000 Fahrenheita
